elevate-cpanel

```diff
@@ -216,6 +217,7 @@ BEGIN {    # Suppress load of all of these at earliest point.
       JetBackup
       NICs
       EA4
+      BootKernel
```

Comment:
This list indicates the order blockers will be checked BTW

Reply:
Yes, and I (think I) want this blocker to be checked after package updates but before the Linode GRUB workaround. Is there another consideration that I might have missed?